doc/languages-frameworks/python.section.md

```diff
@@ -1270,6 +1270,7 @@ are used in `buildPythonPackage`.
   with the `pipInstallHook`.
 - `pythonRelaxDepsHook` will relax Python dependencies restrictions for the package.
   See [example usage](#using-pythonrelaxdepshook).
+- `unittestCheckHook` will run tests with `python -m unittest discover`.
```

Comment:
Should this include an example (showing how to set flags?) like `pytestCheckHook`?

Reply:
sure, why not

Reply:
Had a go at it, let me know what you think.